web/toDoList.jsp

```diff
@@ -1,160 +1,147 @@
-                    listEmpty = true;
-                    $('#emptyListMessage').css("display", "block");
+</div>
+<script>
+    $(function () {
+        $('.datetimepicker').datetimepicker({});
+        var timeLeftMessageFunction = function (id) {
+            $(id).click(function () {
+                var resetPanel = $("#resetPanel");
+                resetPanel.insertAfter(this);
+                var timeLeft = $(this).html();
+                $("#message").html("time remaining: " +timeLeft.split(":").pop());
+                resetPanel.slideToggle();
+                $("#editButton").click(function () {
+                })
+            })
+        };
+        $(".timeLeftMessageStyle").each(function () {
+            var idOfTimeLeftMessage = '#' + this.id;
+            timeLeftMessageFunction(idOfTimeLeftMessage);
+        });
+        $(".remainderButtonStyle").each(function () {
+            var idOfRemainder = '#' + this.id;
+            reminderFunction(idOfRemainder);
+        });
+        var listEmpty = <%=isListEmpty%>;
+        var idOfLastItem = 0;
+        $(".checkboxStyle").css('visibility', 'hidden');
+        var itemInputNode = $('#todoItemName');
+        itemInputNode.keypress(function (e) {
+            if (e.which == 13) {
+                var itemName = itemInputNode.val();
+                if (!itemName.trim()) {
+                    return;
                 }
+                $('#emptyListMessage').css("display", "none");
+                $.ajax({
+                    url: 'addItem.jsp',
+                    data: {
+                        item: itemName,
+                    },
+                    method: 'POST'
+                }).done(function (data) {
+                    idOfLastItem = parseInt(data);
+                    if (listEmpty) {
+                        listEmpty = false;
+                        $('#listBody').prepend("<div class='toDoListDataStyle' id='listItemId-" + idOfLastItem + "'><span contenteditable='true' id='item-" + idOfLastItem + "' class ='itemStyle'>" + itemName + " </span><span class='checkboxStyle' id='checkboxId-" + idOfLastItem + "' ><input type='checkbox'/></span><span id='remainderButton-" + idOfLastItem + "' class='button btn btn-default btn-sm remainderButtonStyle'>Reminder</span></div>");
+                        $('#checkboxId-' + idOfLastItem).css('visibility', 'hidden');
+                    } else {
+                        $('#listBody').last().append("<div class='toDoListDataStyle' id='listItemId-" + idOfLastItem + "'> <span contenteditable='true' id='item-" + idOfLastItem + "' class ='itemStyle'>" + itemName + " </span><span class='checkboxStyle' id='checkboxId-" + idOfLastItem + "' ><input type='checkbox'/></span><span id='remainderButton-" + idOfLastItem + "' class='button btn btn-default btn-sm remainderButtonStyle'>Reminder</span></div>");
+                        $('#checkboxId-' + idOfLastItem).css('visibility', 'hidden');
```

Comment:
instead of setting visibility hidden after creating the element, set it as a style of the `<span class='checkboxStyle' id='checkboxId-" + idOfLastItem + "' >`